src/js/components/RadioButton/doc.js

```diff
@@ -97,6 +97,11 @@ export const themeDoc = {
     type: 'string',
     defaultValue: 'small',
   },
+  'radioButton.fontWeight': {
```

Comment:
```suggestion
  'radioButton.font.weight': {
```
I'm inclining to support the structure of  `radioButton.font.weight` instead of `fontWeight`, and the reason is that if we would like to scale in the future and enhance the font object (e.g. adding font family) it will be cleaner and more straight forward. thoughts? @ericsoderberghp I would like your quick feedback here as well.

Reply:
I do like that better - @ericsoderberghp let me know your thoughts.

Reply:
@ericsoderberghp  approves :), @jackjwilliams let's head this direction, thank you!